app/views/Projects/edit.html

```diff
@@ -41,7 +41,7 @@
         <div class="controls">
             <input name="url" value="${flash.url ?: project.url}" maxlength="${util.Util.VARCHAR_SIZE}"/>
             <span class="help-inline">​#{error 'url'/}</span>
```

Comment:
This isn’t related to the pull request, but in the Git diff, I saw that there’s a zero-width space (U+200B) in this line between the `>` and the `#`. Same thing in `claimForm.html` just above. Is this intentional?